Bulldozer.java

```diff
@@ -0,0 +1,18 @@
+package org.example;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+
+            System.out.println("Bulldozeer started its work.");
+

```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present